helm/ovn-kubernetes/values.yaml

```diff
@@ -51,7 +51,7 @@ global:
   # -- A comma separated set of IP subnets and the associated hostsubnetlengths (eg, \"10.128.0.0/14/23,10.0.0.0/14/23\") to use with the extended hybrid network
   hybridOverlayNetCidr: ""
   # -- Whether or not to use Admin Network Policy CRD feature with ovn-kubernetes
-  enableAdminNetworkPolicy: true
+  enableAdminNetworkPolicy: false
```

Comment:
hmm you have a setting for this and you are changing it to false, but in create_ovn_kubernetes you are setting it to true. What am I missing?

Reply:
Right. Sorry for the confusion. In order to `enableAdminNetworkPolicy`, we need to explicitly download the CRDs. My thinking here is that by default, we should not require it and simply have it disabled. However, you correctly pointed out that in `kind-helm.sh`, we are explicitly enabling it after grabbing the CRD. There is also a `basic_deploy.sh` script in [this repository](https://github.com/ovn-org/ovn-kubernetes/blob/master/helm/basic-deploy.sh) which will benefit from this being disabled by default since it does the bare minimum Helm deploy.